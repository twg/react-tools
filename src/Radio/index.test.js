/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import { Radio } from './'

const noop = () => {}
describe('Radio', () => {
  it('renders an unchecked radio button', () => {
    let component = renderer.create(
      <Radio checked={false} id="unchecked" label="Unchecked Example" />
    )

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('renders an checked radio button', () => {
    let component = renderer.create(
      <Radio checked={true} id="checked" label="Checked Example" />
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
