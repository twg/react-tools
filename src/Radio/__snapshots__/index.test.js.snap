// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio renders an checked radio button 1`] = `
<div
  className="wrapper"
>
  <label
    className="label"
    htmlFor="checked"
  >
    <input
      checked={true}
      className="radio"
      name={undefined}
      onChange={undefined}
      type="radio"
    />
    <span
      className="span"
    />
  </label>
  <p
    style={
      Object {
        "marginTop": "4px",
      }
    }
  >
    Checked Example
  </p>
</div>
`;

exports[`Radio renders an unchecked radio button 1`] = `
<div
  className="wrapper"
>
  <label
    className="label"
    htmlFor="unchecked"
  >
    <input
      checked={false}
      className="radio"
      name={undefined}
      onChange={undefined}
      type="radio"
    />
    <span
      className="span"
    />
  </label>
  <p
    style={
      Object {
        "marginTop": "4px",
      }
    }
  >
    Unchecked Example
  </p>
</div>
`;
